app/views/items/index.html.haml

```diff
@@ -46,7 +46,8 @@
 
   .row
     .column#menu
-      -@categories.each_value do |category_items|
+      -@categories.each do |category, category_items|
+        %h2{ style: "text-align: center;margin-top: 1em; "}= category.capitalize
```

Comment:
We tend to avoid putting styles directly onto tags in our HTML. We prefer to use CSS classes. How those classes should get named and organized is a better question for one of our designers. We've gone through a couple iterations of paradigms, so I'm not sure what's the current standard. But just something like `.category-title` would suffice for something like this.